osu.Game/Screens/Play/HUD/GameplayAccuracyCounter.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public abstract class GameplayAccuracyCounter : PercentageCounter
+    {
+        [BackgroundDependencyLoader]
+        private void load(ScoreProcessor scoreProcessor)
+        {
+            Current.BindTo(scoreProcessor.Accuracy);
+        }
+    }
```

Comment:
Nit: Not sure about having this existing for accuracy but not existing for other elements like combo.

I think base skinnable elements classes should only exist for elements that share the same logic (e.g. `GameplayScoreCounter`'s required display digits logic), excluding the data source bind, which is essentially just one line.

Reply:
Reasoning?

Reply:
To reword what I mean:

I think base skinnable elements classes should either only exist for elements that share further logic than just data source bind, or exist for all elements.

Currently, it looks to be inconsistent, which is the reason I brought this up.

But looking back at the example I mentioned above (combo), it seems `LegacyComboCounter` and `DefaultComboCounter` cannot have a base class and would need a bit of refactoring to do so, so I think it's fine to leave this as-is, not worth the time or hassle.

Reply:
Resolving this - as stated above, combo counters can't have a shared base class, since one doesn't make sense there. Heck, the legacy counter is kind of two counters at once, if you think about it. No need to try and fit a round peg into a square hole.

Reply:
Yep, it's fine for now.